### PR TITLE
Added documentation for choice_translation_domain option

### PR DESCRIPTION
**Pull request by peterrehm**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes #13651 |
| Applies to | Symfony 2.7+ |
| Fixed tickets | - |

### CODE REVIEW

File: reference/forms/types/options/choice_translation_domain.rst.inc [hunk 1]
```diff
@@ -0,0 +1,15 @@
+choice_translation_domain
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 2.7
```
Comment (javiereguiluz) [inline reference/forms/types/options/choice_translation_domain.rst.inc]: According to the original PR, this option was added in Symfony 2.8: https://github.com/symfony/symfony/commit/3909460

Comment (javiereguiluz) [inline reference/forms/types/options/choice_translation_domain.rst.inc]: Forget my previous comment. In 2.8 we added support for this option in date-related types.

**Comment by peterrehm** [comment]
Any issues with this? Is this good to go?

File: reference/forms/types/choice.rst [hunk 2]
```diff
@@ -346,29 +348,32 @@ type:
...
++----------------------------+--------------+-------------------------------------------------------------------+
+| choices                    | ``array``    | A nested array containing the ``ChoiceView`` objects of           |
+|                            |              | the remaining choices.                                            |
++----------------------------+--------------+-------------------------------------------------------------------+
+| separator                  | ``string``   | The separator to use between choice groups.                       |
++----------------------------+--------------+-------------------------------------------------------------------+
+| placeholder                | ``mixed``    | The empty value if not already in the list, otherwise             |
+|                            |              | ``null``.                                                         |
++----------------------------+--------------+-------------------------------------------------------------------+
+| choice_translation_domain  | ``mixed``    | ``boolean``, ``null`` or ``string`` to determine if value should  |
```
Comment (xabbuh) [inline reference/forms/types/choice.rst]: [...] if the value [...]

File: reference/forms/types/options/choice_translation_domain.rst.inc [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    The ``choice_translation_domain`` option was introduced in Symfony 2.7.
+
+**type**: ``string``, ``boolean`` or ``null``
+
+This option determines if the choice values should be translated and in which
+translation domain.
+
+The values of the choice_translation_domain option can be true (reuse the current
```
Comment (xabbuh) [inline reference/forms/types/options/choice_translation_domain.rst.inc]: `choice_translation_domain` should be enclosed by double backticks.

Comment (xabbuh) [inline reference/forms/types/options/choice_translation_domain.rst.inc]: the same applies to `true`

File: reference/forms/types/options/choice_translation_domain.rst.inc [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+translation domain), false (disable translation), null (uses the parent translation
```
Comment (xabbuh) [inline reference/forms/types/options/choice_translation_domain.rst.inc]: and to `false` and `null`

File: reference/forms/types/options/choice_translation_domain.rst.inc [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+domain or the default domain) and a string which represents the exact translation
```
Comment (xabbuh) [inline reference/forms/types/options/choice_translation_domain.rst.inc]: [...] **or** a string [...]

**Comment by xabbuh** [comment]
@peterrehm I have left some minor comments. Besides that this looks ready to be merged.

**Comment by WouterJ** [comment]
:+1:

**Comment by xabbuh** [comment]
Thank you @peterrehm.

**Pull request merged**